proseco/tests/test_catalog.py

```diff
@@ -242,6 +241,71 @@ def test_big_sim_offset():
     assert all(name in aca.fids.colnames for name in names)
 
 
+def test_calling_with_t_ccd():
+    """Test that calling get_aca_catalog with t_ccd arg sets all the
+    CCD attributes correctly in the nominal case of a temperature
+    below the planning limit.
+
+    """
+    dark = DARK40.copy()
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(mag=8.0, n_stars=8)
+    t_ccd = ACA.aca_t_ccd_planning_limit - 1.0
+
+    kwargs = mod_std_info(stars=stars, dark=dark, t_ccd=t_ccd)
+    aca = get_aca_catalog(**kwargs)
+    assert np.isclose(aca.t_ccd, t_ccd)
+    assert np.isclose(aca.t_ccd_acq, t_ccd)
+    assert np.isclose(aca.t_ccd_guide, t_ccd)
+
+    assert np.isclose(aca.t_ccd_eff_acq, t_ccd)
+    assert np.isclose(aca.t_ccd_eff_guide, t_ccd)
+
+    assert np.isclose(aca.acqs.t_ccd, t_ccd)
+    assert np.isclose(aca.guides.t_ccd, t_ccd)
+
+
+t_ccd_cases = [(-0.5, 0, 0),
+               (0, 0, 0),
+               (0.5, 1.5, 1.4)]
+
+
+@pytest.mark.parametrize('t_ccd_case', t_ccd_cases)
+def test_t_ccd_effective_acq_guide(t_ccd_case):
+    """Test setting of effective T_ccd temperatures for cases above and
+    below the planning limit.
+
+    """
+    dark = DARK40.copy()
+    stars = StarsTable.empty()
+    stars.add_fake_constellation(mag=8.0, n_stars=8)
+
+    t_limit = ACA.aca_t_ccd_planning_limit
+
+    t_offset, t_penalty_acq, t_penalty_guide = t_ccd_case
+    # Set acq and guide temperatures different
+    t_ccd_acq = t_limit + t_offset
+    t_ccd_guide = t_ccd_acq - 0.1
+
+    kwargs = mod_std_info(stars=stars, dark=dark,
+                          t_ccd_acq=t_ccd_acq, t_ccd_guide=t_ccd_guide)
```

Comment:
I think maybe your tests were passing because you were passing t_ccd from STD_INFO *and* t_ccd_acq and t_ccd_guide?